service/create/pki.go

```diff
@@ -0,0 +1,119 @@
+package create
+
+import (
+	"strings"
+
+	microerror "github.com/giantswarm/microkit/error"
+
+	"github.com/giantswarm/certctl/service/pki"
+	"github.com/giantswarm/certctl/service/token"
+)
+
+const (
+	// TODO Choose a sensible value and move to ConfigMap for V1
+	CertificateAuthorityTTL = "1440h"
+)
+
+func (s *Service) checkPKIBackend(clusterID string) bool {
+	service, err := s.getPKIService()
+	if err != nil {
+		return false
+	}
+
+	// Check PKI config.
+	mounted, err := service.IsMounted(clusterID)
+	if !mounted || err != nil {
```

Comment:
Lets please have always proper error handling. When shit hits the fan we are really screwed by this code because we do not see anything.